### ISSUE TITLE
comrak build fails on windows

### ISSUE THREAD

**Issue by Lioncat2002** [kivikakk/comrak#226]
I am trying to use the latest version of comrak for my project [Ferry](https://github.com/Lioncat2002/Ferry) <br>
but comrak fails to build on windows 
```
 Compiling comrak v0.13.1
error[E0463]: can't find crate for `syntect`
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\lib.rs:85:1
   |
85 | extern crate syntect;
   | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

error[E0433]: failed to resolve: use of undeclared type `SyntaxSet`
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:24:25
   |
24 |             syntax_set: SyntaxSet::load_defaults_newlines(),
   |                         ^^^^^^^^^ use of undeclared type `SyntaxSet`

error[E0433]: failed to resolve: use of undeclared type `ThemeSet`
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:25:24
   |
25 |             theme_set: ThemeSet::load_defaults(),
   |                        ^^^^^^^^ use of undeclared type `ThemeSet`

error[E0412]: cannot find type `SyntaxSet` in this scope
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:15:17
   |
15 |     syntax_set: SyntaxSet,
   |                 ^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `ThemeSet` in this scope
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:16:16
   |
16 |     theme_set: ThemeSet,
   |                ^^^^^^^^ not found in this scope

error[E0425]: cannot find function `highlighted_html_for_string` in this scope
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:31:9
   |
31 |         highlighted_html_for_string(
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `highlighted_html_for_string` in this scope
  --> C:\Users\Msi\.cargo\registry\src\github.com-1ecc6299db9ec823\comrak-0.13.1\src\plugins\syntect.rs:72:29
   |
72 |         self.remove_pre_tag(highlighted_html_for_string(
   |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

   Compiling clap v3.1.2
Some errors have detailed explanations: E0412, E0425, E0433, E0463.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `comrak` due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```
eventhough it builds fine on linux

rustc version for windows: `1.59.0`

rustc version for linux `1.59.0`

**Comment by kivikakk** [comment]
Thanks for the report. This should be fixed in [Comrak 0.13.2](https://crates.io/crates/comrak/0.13.2). Please let me know if this doesn't work for you.

**Comment by Lioncat2002** [comment]
It's working now
Thank you!
